src/router.js

```diff
@@ -1,53 +1,48 @@
-import Vue from 'vue'
-import Router from 'vue-router'
+import Vue from "vue";
+import Router from "vue-router";
 
-import Welcome from './views/Welcome'
-import SignUp from './views/SignUp'
-import Discover from './views/Discover'
-import EventPage from './views/EventPage'
+// import Welcome from "./views/Welcome";
+import SignUp from "./views/SignUp";
+import Discover from "./views/Discover";
+import EventPage from "./views/EventPage";
 
 /* FOR TESTING */
-import createEventModal from './components/createEventModal'
-import eventInitiation from './components/eventInitiation'
+import createEventModal from "./components/createEventModal";
+// import eventInitiation from "./components/eventInitiation";
+import EventInitiation from "./views/EventInitiation.vue";
 
-
-Vue.use(Router)
+Vue.use(Router);
 
 const router = new Router({
-  mode: 'history',
+  mode: "history",
   base: process.env.BASE_URL,
   routes: [
     {
-      path: '/',
-      name: 'onboarding',
-      component: Welcome,
-    },
-    {
-      path: '/signup',
-      name: 'sign-up',
-      component: SignUp,
+      path: "/",
+      name: "discover",
+      component: Discover
     },
     {
-      path: '/discover',
-      name: 'discover',
-      component: Discover
+      path: "/signup",
+      name: "sign-up",
+      component: SignUp
     },
     {
```

Comment:
I think you accidentally removed sign-in page route. This is also kindof a breaking change so would suggest putting in PR desc

Reply:
The buttons in the explore bar still redirect to 'discover', so will need to change that so that it directs to the correct route

Reply:
i think i mentioned that we'll be redoing the onboarding, as it might not be good enough? i'm thinking about redesigning it and making it fit our new theme more.
about the DISCOVER button, isn't redirecting to `/discover` correct?

Reply:
Might still wanna keep the route that leads to the signup page though, otherwise we'd never be able to access it - I made responsiveness changes to each of these pages as well so they're quite good to go on the other PR.
Also you removed the `/discover` path and changed it to the default `/` a few lines above
```suggestion
    {	
      path: '/welcome',	
      name: 'Welcome,	
      component: Welcome,	
    },
    {
```

Reply:
ok sure made changes